src/commands/templates/install.js

```diff
@@ -15,13 +15,38 @@ const { runScript } = require('../../lib/helper')
 const { writeObjectToPackageJson, readPackageJson, getNpmDependency, processNpmPackageSpec, TEMPLATE_PACKAGE_JSON_KEY } = require('../../lib/npm-helper')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app-templates:templates:install', { provider: 'debug' })
 
+// aio-lib-console-project-installation depenencies
+const path = require('path')
+const sdk = require('@adobe/aio-lib-console')
+const loadConfig = require('@adobe/aio-cli-lib-app-config')
+const templateHandler = require('@adobe/aio-lib-console-project-installation')
+
 class InstallCommand extends BaseCommand {
   async run () {
     const { args } = this.parse(InstallCommand)
     let templateName
 
     await runScript('npm', process.cwd(), ['install', args.path])
 
+    // Integrate with the template installation library
+    // Instantiate Adobe Developer Console SDK
+    await this.login()
+    const apiKey = 'aio-cli-console-auth'
+    const client = await sdk.init(this.accessToken, apiKey)
+
+    // Instantiate App Builder Template Manager
+    const installConfigFile = path.join(
+      process.cwd(),
+      'node_modules',
+      args.path,
+      'install.yml'
+    )
+    const templateManager = templateHandler.init(client, installConfigFile)
+    const appConfig = loadConfig({})
+    const orgId = appConfig.aio.project.org.id
```

Comment:
@roribio @slitviachenko users may want to be able to select the organization and project in the CLI (see https://github.com/adobe/aio-cli-plugin-app/blob/fb4b9ca59e484bc1f8dad3f6ebe149886aa5b856/src/commands/app/init.js#L99). a future improvement, maybe?

Reply:
Yes, definitely a future improvement 👍 Good idea though.